test/Libraries/RevitIntegrationTests/WorkflowTest.cs

```diff
@@ -198,12 +206,8 @@ public void Test_PanelWall()
 
             //check AdaptiveComponent.ByPoint
             var acID = "85e957b8-90b7-46a5-823c-18aaa4155c98";
-            AssertPreviewCount(acID, 60);
```

Comment:
This does not reflect the real situation in the graph. Here is should be a single element.